interface/src/app/sample/sampleMocks.spec.js

```diff
@@ -0,0 +1,97 @@
+angular.module('adage.mocks.sample', [])
```

Comment:
@dongbohu: it sounds as if you have a Vision for how we should be doing testing. I look forward to learning more as we follow up via issue #81.